index.d.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference types="node" />
```

Comment:
this is the format for referencing another type, in this case `@types/node`? new to me.

Reply:
It's so that TS knows that `inspector` comes from `node` - it's not a separate module